kroxylicious-operator/install/01.ClusterRole.kroxylicious-operator-watched.yaml

```diff
@@ -28,13 +29,26 @@ rules:
       # - create
       # - patch
       # - update
-  - apiGroups:
+  - # The operator needs to update the status on its own CRs
+    apiGroups:
       - "kroxylicious.io"
     resources:
       - kafkaproxies/status
       - kafkaservices/status
       - kafkaproxyingresses/status
+      - kafkaprotocolfilters/status
```

Comment:
kafkaprotocolfilters is part of the apiGroups filter.kroxylicious.io.

Reply:
You're right. But I wonder whether we still want that, given how we've moved away from CRD-per-filter.

Reply:
Yes, I wondered whether the use of two groups had any justification now.   Happy to see it collapsed to a single group.